toml.md

```diff
@@ -36,10 +36,15 @@ should be easy to parse into data structures in a wide variety of languages.
 
 ## Spec
 
+A TOML file must be a valid UTF-8 encoded Unicode document. Specifically this
+means that, should a file as a whole not form a
+[well-formed code-unit sequence](https://unicode.org/glossary/#well_formed_code_unit_sequence),
+the file must be rejected (preferably) or ill-formed byte sequences must be
+replaced with U+FFFD as per the Unicode spec.
```

Comment:
Nit: I think we should not use abbreviations like these, I’d prefer “As per the Unicode Specification”. We could even link to the specification’s homepage (though, to be fair, 25 years in, most people will just know).

Reply:
We're linking deep into unicode.org, and throughout the document, we say "Unicode" thirteen times. We mention no other character mapping mechanism or encoding, except for ASCII, and then only when describing numbers. How on earth will this get confused with any other standard?

The mention of ASCII alone has got my dander up.

Reply:
Lol. Yeah, leave the link. The nit was just about “spec” vs “specification” mainly, really ;).

Reply:
Oh, just that? Sure, we could do that. I have no objections to "Unicode specification" instead of "Unicode spec". But if we change it once in one place, how about we change it in three other places where the abbrev "spec" is used (referring to the first section, which is more "preliminaries" than "spec" anyway)? I'm willing to do that for the sake of formality. But I doubt that is reason enough to make such a change across the board.

So what do you say to one instance of "specification" and three of "preliminaries", all properly cased, to replace the abbreviation "spec"?